br/pkg/restore/snap_client/client.go

```diff
@@ -948,7 +949,25 @@ func (rc *SnapClient) ExecDDLs(ctx context.Context, ddlJobs []*model.Job) error
 	return nil
 }
 
-func (rc *SnapClient) ResetSpeedLimit(ctx context.Context) error {
+func (rc *SnapClient) resetSpeedLimit(ctx context.Context) {
```

Comment:
curious why do we need two parameters to control rate limit, we can set the rate limit to be 0 to effectively turning it off. maybe can remove the `hasSpeedLimited` ?

just a nit, we probably could keep the naming consistent, both using word containing `ratelimit` instead of `speed` could be convenient for the full text search in the code base.